package-overrides/github/twbs/bootstrap@3.3.7.json

```diff
@@ -0,0 +1,24 @@
+{
+  "main": "js/bootstrap",
+  "shim": {
+    "js/bootstrap": {
+      "deps": ["jquery"],
+      "exports": "$"
+    }
+  },
+  "dependencies": {
+    "jquery": "*"
+  },
+  "files": ["dist", "fonts", "js", "css", "less", "grunt", "LICENSE"],
+  "systemjs": {
```

Comment:
This is a jspm-0.17 specific property.

For jspm 0.16 compatibility we should still use:

```
"shim": {
  "dist/js/bootstrap": {
    "deps": ["jquery"],
    "exports": "$"
  }
}
```